extension/content.css

```diff
@@ -1,16 +1,15 @@
-mark {
-}
 .markedYellow {
-	background-color:yellow!important;
+	background-color: yellow;
 }
 .markedBlue {
-	background-color:#c9daf8 !important; /* Same shade of blue as when normally highlight in browser */
+	/* Same shade of blue as when normally highlight in browser */
+	background-color: rgb(201, 218, 248);
 }
 .markedLightBlue {	
-	background-color:#e6effd !important; /* Lighter blue */
+	background-color: rgb(230, 239, 253); /* Lighter blue */
```

Comment:
Changed to rgb syntax cos it helped me debug previously.
If you have a strong pref for the # syntax lmk and ill revert.

Reply:
Nope I actually prefer rgb as well! Easier to move between rgba for experimenting with opacity too

Reply:
LGTM